dbcon/mysql/ha_mcs.cpp

```diff
@@ -16,6 +16,14 @@
    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
    MA 02110-1301, USA. */
 
+// This makes specific MDB classes' attributes public to implement
+// MCOL-4740 temporary solution. Search for MCOL-4740
+// to get the actual place where it is used.
+#define updated_leaves \
```

Comment:
I have so many words to say, I'd better no

Reply:
I know, I know but it is better to save the precious breath )